Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'doorkeeper'
 gem 'faraday'
 gem 'finite_machine'
 gem 'govuk_notify_rails', '~> 2.1.2'
+gem 'hiredis', '~> 0.6.3'
```

Comment:
Not come across this gem before.. reading through the docs I'm not immediately sure why it's needed? https://github.com/redis/hiredis-rb

Reply:
It was suggested in the rails caching guide: go faster stripes for Redis - https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore

Reply:
Cool - fast is good 👍